Source/Core/BoxGeometry.js

```diff
@@ -111,14 +128,51 @@ define([
         var max = corner;
 
         var newOptions = {
-            minimumCorner : min,
-            maximumCorner : max,
+            minimum : min,
+            maximum : max,
             vertexFormat : options.vertexFormat
         };
         return new BoxGeometry(newOptions);
     };
 
     /**
+     * Creates a cube from the dimensions of an AxisAlignedBoundingBox.
+     *
+     * @param {AxisAlignedBoundingBox} boundingBox A description of the AxisAlignedBoundingBox.
+     * @returns {BoxGeometry}
+     *
+     * @exception {DeveloperError} AxisAlignedBoundingBox must be defined.
+     *
+     * @see BoxGeometry.createGeometry
+     *
+     * @example
+     * var aabb = Cesium.AxisAlignedBoundingBox.fromPoints(Cesium.Cartesian3.fromDegreesArray([
+     *      -72.0, 40.0,
+     *      -70.0, 35.0,
+     *      -75.0, 30.0,
+     *      -70.0, 30.0,
+     *      -68.0, 40.0
+     * ]));
+     * var box = Cesium.BoxGeometry.fromAxisAlignedBoundingBox({
+     *      boundingBox: aabb
+     * });
+     */
+    BoxGeometry.fromAxisAlignedBoundingBox = function(boundingBox) {
+        if (!defined(boundingBox)) {
+            throw new DeveloperError('boundingBox is required.');
+        }
+
+        var min = boundingBox.minimum;
```

Comment:
Since we only use them in one place, there's no need to declare these locals.  Let's keep it concise:
```javascript
return new BoxGeometry({
    minimum : boundingBox.minimum,
    maximum : boundingBox.maximum
});
```